packages/evm-connector/src/generated/predicates/0xca07380d3c0d9890519ee727bf522eaf19e2acefab240c9a22b0aa90f41c87d6/index.ts

```diff
@@ -0,0 +1,31 @@
+/* Autogenerated file. Do not edit manually. */
+
+export const generationDate = 1722897433097;
+// biome-ignore lint: Autogenerated file
```

Comment:
this comment is not needed

Reply:
also remove it from other packages

Reply:
just to document: this comment is left as is because the file was formatted by me. When it is autogenerated, the file is like the following:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/34346675-8a9e-4e99-8052-fcc0ce946f9b">